saleor/plugins/admin_email/plugin.py

```diff
@@ -97,6 +111,16 @@ class AdminEmailPlugin(BasePlugin):
     ] + DEFAULT_EMAIL_CONFIGURATION  # type: ignore
 
     CONFIG_STRUCTURE = {
+        constants.STAFF_PASSWORD_RESET_SUBJECT_FIELD: {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": DEFAULT_SUBJECT_HELP_TEXT,
+            "label": "Staff reset password subject",
```

Comment:
Do we have to say "staff" in these labels? It's an admin email plugin, so all these emails would be sent to dashboard users. It should be mentioned in the plugin description.

Reply:
Maybe we could change it simply to "Reset password email subject"